packages/singleton-manager/demo/singleton-complex/overlayCompatibility.js

```diff
@@ -0,0 +1,64 @@
+import { OverlaysManager } from 'overlays';
```

Comment:
As this compatibility file is located in the root of the app, is it the goal/guideline that the app (always) provides it for every singleton? Or only for singletons that have clashes?

I imagine that a v2 of a feature could also provide the compatibility layer?

Reply:
Only for singletons that have clashes I guess. I do think it's our responsibility as maintainers of lion provide compatibility layers that our users can import, so we do some of the work for them already. 

So we would export based on 3 hypothetically breaking majors:
- OverlaysManager__OneTwo
- OverlaysManager__TwoOne
- OverlaysManager__ThreeOne
- OverlaysManager__OneThree

Reply:
We can start with providing these compatibilities once we have 1.x